modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -4616,16 +4623,10 @@ public CacheGroupDescriptor cacheGroupDescriptor(int grpId) {
      * @return Cache descriptor.
      */
     public @Nullable DynamicCacheDescriptor cacheDescriptor(int cacheId) {
-        for (DynamicCacheDescriptor cacheDesc : cacheDescriptors().values()) {
-            CacheConfiguration ccfg = cacheDesc.cacheConfiguration();
-
-            assert ccfg != null : cacheDesc;
+        Map<Integer, DynamicCacheDescriptor> cacheIdToNameMapping = cacheIdToNameMapping();
```

Comment:
I think it's too verbose for such a simple method. `return cachesInfo.registeredCachesById().get(cacheId);` or something like that will be more readable.
`cacheIdToNameMapping()` method in this class is redundant, it used only once and only brings extra complexity to code navigation.